packages/discovery-react-components/src/components/CIDocument/components/CIDocument/__tests__/CIDocument.spec.tsx

```diff
@@ -48,22 +56,36 @@ describe('<CIDocument />', () => {
       globalGetByText(filters, 'Currency');
     });
 
-    it('selects Relations and checks details panel', async () => {
-      const tabButton = await waitForElement(() => {
-        const tabs = getByTestId('document-info-tabs');
-        return globalGetByText(tabs, 'Relations');
-      });
-      fireEvent.click(tabButton);
+    it('loads the correct tabs and pane sections titles', async () => {
+      // Attribute and Relations tabs exist
+      const attributesTab = getByTestId('attributes-tab');
+      const relationsTab = getByTestId('relations-tab');
+      expect(attributesTab.textContent).toEqual('Attributes');
+      expect(relationsTab.textContent).toEqual('Relations');
```

Comment:
How hard is it to also add a test that filters and metadata tabs _do not_ exist?